bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/types/HSBType.java

```diff
@@ -71,9 +71,12 @@ public HSBType(String value) {
         if (value != null) {
             String[] constituents = value.split(",");
```

Comment:
As every part should be trimmed, why not using something like this?

```Java
List<String> constituents = Arrays.stream(value.split(",")).map(in -> in.trim()).collect(Collectors.toList());
```

Reply:
Alright, I have changed it to make use of the stream API.

Reply:
Makes sense. Done.